examples/getting_started/example_dataset_config_test.yaml

```diff
@@ -0,0 +1,18 @@
+settings:
+  src_path: data/raw/redpajama_v2_samples_512_test.jsonl
+  dst_path: data/mem_map/redpajama_v2_samples_512_test.pbin
+  index_path: data/mem_map/redpajama_v2_samples_512_test.idx
+  jq_pattern: .raw_content
+  num_cpus: ${node_env:num_cpus}
+  eod_token: <|endoftext|>
+  processing_batch_size: 1000
+  raw_samples_queue_size: 300
+  processed_samples_queue_size: 300
+
+tokenizer:
+  component_key: tokenizer
+  variant_key: pretrained_hf_tokenizer
+  config:
+    pretrained_model_name_or_path: tokenizer
+    padding: false
+    max_length: 1024
```

Comment:
maybe 512?

Reply:
Fixed in e391cfc2646dcb52e9943449cc42f665f3b3b5fc.